README.md

```diff
@@ -32,7 +32,15 @@ In practice, the Lengths vector is stored in the form of Offsets to reduce compl
 
 ## Usage
 The synthetic dataset provided in this project serves as sample inputs for the Indices and Offsets vectors;
-the corresponding Lengths vector is provided for correctness validation as well.
+the corresponding Lengths vector is provided for correctness validation as well. Each `pt` file contains an
```

Comment:
Looking forward, could this be rewritten along the lines of:
```
... as well. Each `pt` file in the `embedding_bag` directory contains ... with a batch size and number of tables specified in the filename.  For example, the dataset `fbgemm_t856_bs65536.pt` represents a single batch of 65536 samples for 856 tables.
```

Reply:
Good point. Updated.